com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.cs

```diff
@@ -184,7 +184,8 @@ internal void AddRequestsToLightmapper()
 
         private void RemoveRequestsFromLightmapper()
         {
-            UnityEditor.Experimental.Lightmapping.SetAdditionalBakedProbes(s_BakingID, null);
+            if (RenderPipelineManager.currentPipeline != null)
+                UnityEditor.Experimental.Lightmapping.SetAdditionalBakedProbes(s_BakingID, null);
```

Comment:
@FrancescoC-unity fyi added that check to prevent the traditional error when opening a project with APV. That would make the tests fail